using_images/s2i_images/php.adoc

```diff
@@ -143,3 +143,6 @@ If the `*DocumentRoot*` of the application is nested in the source directory
 the default Apache behavior and specify how application requests should be
 handled. The *_.htaccess_* file must be located at the root of the application
 source.
+
+== Logs
+This image logs primarily to standard out and as such the logs can be viewed via the `*oc logs*` command.  Access logs are stored in *_/tmp/access_log_* which can be viewed use `*oc exec*` to access the container.
```

Comment:
s/use/using/

Maybe also sync up on "via" or "using" in the two sentences (would suggest "using").